src/analytics/src/main/scala/osmesa/analytics/oneoffs/ChangesetStatsCreator.scala

```diff
@@ -111,17 +111,17 @@ object ChangesetStatsCreator
                        'tags.getItem("created_by") as 'editor,
                        'uid,
                        'user,
-                       'created_at,
+                       'createdAt,
```

Comment:
Capitalization depends on whether an `osm2orc` changeset file or a replication-sourced DF is the source here (sorry, that's my bad).

`osm2orc` uses underscores for parity with the changeset XML. Perhaps `ChangesetSource` should follow suit? Or we should have a function that normalizes column names to use before queries like this one.

Reply:
Let's normalize the column names.

Reply:
I'm just going to assume that we're using the osm2orc naming structure (partly because of `model.Changeset`).  If that is going to get us in trouble elsewhere, speak now or forever holster your piece.

Reply:
osm2orc doesn't match `model.Changeset`.  That hadn't really been a problem before since we never actually cast a `DataFrame` of changesets to `Dataset[Changeset]` and the stream emits `Dataset[Row]` (`DataFrame`) even though it uses `Changeset` internally...

This suggests to me that the normalization function should go underscores -> camel case. Though the counter-argument is that certain output formats (JDBC, for one) lowercase column names before writing.

Reply:
Ugh.

Reply:
As far as the current PR goes, this isn't exactly needed.  For the moment, it seems as if wee could provide, in VP, a method (or two) in `ChangesetSource` that could be used to ensure that the names are all either in camel case or in snake case.  For now, we do this conversion manually ([here](https://github.com/azavea/osmesa/blob/3349bf5c6c3c46c2bebcb86342e176ba1dde7020/src/analytics/src/main/scala/osmesa/analytics/oneoffs/MergeChangesets.scala#L100-L113)), but automatic means for converting should be provided.  I've created an issue [here](https://github.com/geotrellis/vectorpipe/issues/113) to capture this need.

If that's sufficient, then I suggest we just move on and not worry too much about the concern in this comment.  I'll leave a note in the code to clean this up?